docs/cookbook/executing_long_running_console_commands.md

```diff
@@ -60,26 +63,32 @@ To avoid quickly running out of memory while executing such commands you should
     ```
 
 1.  Run PHP without memory limits using: `php -d memory_limit=-1 app/console <command>`
-1.  Disable `xdebug` *(PHP extension to debug/profile php use)* when running the command, this will cause php to use much more memory.
+1.  Disable `xdebug` *(PHP extension to debug/profile php use)* when running the command,
+as it causes PHP to use much more memory.
 
 !!! note "Memory will still grow"
 
-    Even when everything is configured like described above, memory will grow for each iteration of indexing/inserting a content item with at least *1kb* per iteration after the initial first 100 rounds. This is expected behavior; to be able to handle more iterations you will have to do one or several of the following:
+    Even when everything is configured like described above, memory will grow for each iteration
+    of indexing/inserting a Content item with at least *1kb* per iteration after the initial first 100 rounds.
+    This is expected behavior; to be able to handle more iterations you will have to do one or several of the following:
 
     - Change the import/index script in question to [use process forking](#process-forking-with-symfony) to avoid the issue.
     - Upgrade PHP: *newer versions of PHP are typically more memory-efficient.*
     - Run the console command on a machine with more memory (RAM).
 
-### Process forking with Symfony
+## Process forking with Symfony
 
-The recommended way to completely avoid "memory leaks" in PHP in the first place is to use processes, and for console scripts this is typically done using process forking which is quite easy to do with Symfony.
+The recommended way to completely avoid "memory leaks" in PHP in the first place is to use processes,
+and for console scripts this is typically done using process forking which is quite easy to do with Symfony.
```

Comment:
We can easily break them in 2 sentences: The recommended way to completely avoid "memory leaks" in PHP is to use processes. For console scripts this is typically done by using process forking which is quite easily achievable with Symfony.